packages/grid_client/src/primitives/nodes.ts

```diff
@@ -438,7 +441,9 @@ class Nodes {
       ret_count: options.ret_count,
       region: options.region,
     };
-    return Object.entries(params)
+
+    const filteredParams = removeUndefinedAndEmptyString(params);
+    return Object.entries(filteredParams)
```

Comment:
why not extract Object.entries() etc also into the function in the utils? Seems unnecessarily repetitive still